resources/help.txt

```diff
@@ -203,13 +203,16 @@ Options:
 --[init]--
 
 Usage:
+    $appbuilder init
     $appbuilder init [<Command>]
 
-Initializes an existing hybrid or native project for development. You must run the init command with a related command.
+Initializes an existing hybrid, native or mobile website project for development. The command tries to find the project type based on the files in the working folder.
```

Comment:
We should have a final decision on how we're addressing our project types for consistency. We shouldl have either hybrid, native and mobile website, OR Apache Cordova, NativeScript and Mobile Website.

Initializes an existing project for development. 
The command attempts to determine the project type based on the files in the working directory. To specify the project type yourself, run $ appbuilder init <Command>